dev-util/statifier/statifier-1.7.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Statifier is a tool for creating portable, self-containing Linux executables"
+HOMEPAGE="http://statifier.sourceforge.net"
+SRC_URI="http://sourceforge.net/projects/${PN}/files/${PN}/${PV}/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+IUSE=""
+
+RDEPEND="app-shells/bash
+	sys-apps/coreutils
+	virtual/awk"
+
+src_prepare() {
+	# Respect users CFLAGS and LDFLAGS
+	sed -i -e 's/-Wall -O2/$(CFLAGS) $(LDFLAGS)/g' "${S}/src/Makefile" || die
```

Comment:
Nit: please don't use `${S}` when it's the current directory. It confuses whoever's looking at the ebuild as to what the directory actually is ;-).

Reply:
Actually, I'll fix that single thing while testing it. So please don't worry  about it.

Reply:
Thank you! :)